mypyc/lib-rt/native_internal.pyi

```diff
@@ -0,0 +1,14 @@
+class Buffer:
+    def __init__(self, source: bytes | None = None, /) -> None: ...
+    def getvalue(self) -> bytes: ...
+
+def write_bool(data: Buffer, value: bool, /) -> None: ...
```

Comment:
All the read/write functions actually support calling with keywords, e.g. `write_int(data=Buffer(), value=42)` is valid.